### DISCUSSION TITLE
Improving

### DISCUSSION THREAD

**Discussion by alihaidermalik20** [lab-brussels-1/hyf-redesign-group-1#44] [Ideas]
Let's write here about eventual improvements we can make to our webpage such as:
- extra margin or padding looks better somewhere
- an extra feature may be added
- color or background or text changed 
- other animations that can be added etc.

**Comment by alihaidermalik20** [comment]
Our navbar looks amazing. Some suggestions that can later be added maybe:
- When on a small screen, the navbar links may be changed a bit. Such as extra margin and padding on links so 'The Program' is farther away from logo and 'Support us' blue underline isn't so close to 'Apply now' button. Or maybe center the text. Or maybe remove a:hover effect with the underline for small mobile screen because you don't usually hover on your mobile phone but only with a mouse on a computer on a big screen.
![navbar-hyf](https://user-images.githubusercontent.com/95946965/163479182-1157c46a-6150-468f-b930-101a3422746a.png)

- hover effect on the toggler can be improved or removed next week if there is time left. Like instead of gray box shadow when it's hovered over, maybe animation effect can be added to turn the 3 lines into a cross when clicked because usually you will not hover over anything on your phone but click on it directly with the touch screen.
![navbar-hyf-toggler](https://user-images.githubusercontent.com/95946965/163480268-e4cc6d2b-f5db-4b90-b3d3-052ade7f64a7.png)

**Comment by alihaidermalik20** [comment]
font-size of the paragraphs can be made the same?
"We graduated over 100 people in Belgium and more than 300 globally. In Belgium, over 70% found and retained a high-quality job as a developer."
This paragraph has a much larger font-size than the rest.

And maybe make Our Program and Our Impact headers a bit smaller?

And maybe add some top margin on Our Impact to give it space?

'A job or start an advanced.' needs to have 'studies' added to the end.

**Comment by Shewitttt** [comment]
Thank you so much for the reviews @Ali. Really constructive ones :)
